test/python/tokenizer/sanitizers/test_delete_tags.py

```diff
@@ -289,7 +289,7 @@ def run_sanitizer_on(self, country_code, rank_addr, suffix, **kwargs):
 
     def test_string_arguments_pass(self):
         res = self.run_sanitizer_on('de', '25-30', r'[\s\S]*',
-                                    name='foo', ref='foo', name_abc='bar', ref_abc='baz')
+                                    name_xyz='foo', ref_pqr='foo', name_abc='bar', ref_abc='baz')

```

Comment:
Can you please explain why this needed to be changed?

Reply:
This is because of the functionality of `get_filter` function. If we want all suffixes to pass, we should ideally not provide the suffix parameter. If we provide a suffix parameter, it will not allow empty suffixes. This is because the function treats `none` and empty string as different.

Reply:
Right. The `None` case is really hard to wrap your head around. I think for non-programmers it is easier to understand if an absent suffix is handled like the empty string. It would also make the code in `get_filter` easier because the function can go back to accepting `str` only.

Reply:
I agree with this. For these scenarios, we can consider `None` value as an empty string safely. I'll add the changes.